core/crypto/vrf/p256/p256_test.go

```diff
@@ -45,7 +45,7 @@ func TestH2(t *testing.T) {
 			t.Fatalf("Failed generating random message: %v", err)
 		}
 		x := H2(m)
-		if got := len(x.Bytes()); got < 1 && got > l {
+		if got := len(x.Bytes()); got < 1 || got > l {
```

Comment:
`H2` returns a number between 1 and N-1 which means `x.Bytes()` can also range from 1 to 32.